app/components/elements/Popover.tsx

```diff
@@ -0,0 +1,41 @@
+import { Popover as HeadlessPopover, Transition } from "@headlessui/react"
+import { Fragment } from "react"
+
+interface PopoverProps {
+  title: React.ReactNode
+  children: React.ReactNode
+}
+
+export default function Popover({ title, children }: PopoverProps) {
+  return (
+    <HeadlessPopover className="relative">
+      {({ open }) => (
+        <>
+          <HeadlessPopover.Button
+            className={
+              "flex items-center px-1 transition-colors hover:text-red-500 focus:outline-none md:px-2"
+            }
+          >
+            <span>{title}</span>
+            <i className={"fas fa-chevron-down ml-2 h-5 w-5 transition"} />
+          </HeadlessPopover.Button>
+          <Transition
```

Comment:
this is so cool haha

Reply:
The transition? Yeah Headless UI is great!